SimCalorimetry/EcalEBTrigPrimProducers/plugins/EcalEBTrigPrimProducer.cc

```diff
@@ -85,46 +80,37 @@ void EcalEBTrigPrimProducer::beginRun(edm::Run const& run, edm::EventSetup const
 
 unsigned long long EcalEBTrigPrimProducer::getRecords(edm::EventSetup const& setup) {
   // get parameter records for xtals
-  edm::ESHandle<EcalTPGLinearizationConst> theEcalTPGLinearization_handle;
-  setup.get<EcalTPGLinearizationConstRcd>().get(theEcalTPGLinearization_handle);
+  edm::ESHandle<EcalTPGLinearizationConst> theEcalTPGLinearization_handle =
+      setup.getHandle(theEcalTPGLinearization_Token_);
```

Comment:
wouldn't auto work here?
```suggestion
  auto theEcalTPGLinearization_handle = setup.getHandle(theEcalTPGLinearization_Token_);
```
for a more compact impl

Reply:
We kept the type in order to minimize the amount of change done. Using `auto` would definitely work.